### ISSUE TITLE
Convenience Toggles

### ISSUE THREAD

**Issue by mottosso** [pyblish/pyblish-qml#26]
### Goal

Allow for quick access to checking/un-checking validations, based on some logical predicate.

1. Toggle *all*
1. Toggle *none*
1. Toggle *failed*

### Implementation

Draw upon inspiration from Martin Orlowsky's interface design to potentially put buttons next to items in the list of available plug-ins.

![image](https://cloud.githubusercontent.com/assets/2152766/5900940/1e8c9758-a566-11e4-9fd2-8b11c666e7b2.png)

**Comment by tokejepsen** [comment]
With #135, can we close this issue?

**Comment by mottosso** [comment]
I would say no. To toggle All and None still seems quite useful. Maybe worth adding, as an extra item up top/down bottom?

**Comment by tokejepsen** [comment]
What are you expecting to toggle? Is it just validators? All plugins, or instances, or both?

Toggle failed might be a bit trickier, as you can't toggle anything once Pyblish has run and it would include saving the state of the instance/plugins between runs. Unless you had something else it mind?

**Comment by mottosso** [comment]
Yeah, "failed" probably doesn't make sense. But toggling all/none of the instances might make sense?

**Comment by tokejepsen** [comment]
Sure, could do. Question is placement and appearance of a button for it:)

**Comment by mottosso** [comment]
Indeed. :)
